src/Game/NeoServer.Game.Creatures/Model/Players/Player.cs

```diff
@@ -917,6 +917,21 @@ public override ILoot DropLoot()
         return null;
     }
 
+    public void StopAllAction()
+    {
+        if (Attacking)
```

Comment:
I'd remove those ifs and call the 3 stops directly
StopAttack();
StopWalking();
StopFollowing();